.gitignore

```diff
@@ -16,6 +16,7 @@ data/*.minor
 # GNUmakefile deps
 src/.deps/*
 src/corels
+src/tests
```

Comment:
This is good, but I would prefer if we could also move the test code to a separate folder `test/`